docs/tendermint-core/configuration.md

```diff
@@ -473,3 +473,17 @@ Here's a brief summary of the timeouts:
 - `timeout_commit` = how long we wait after committing a block, before starting
   on the new height (this gives us a chance to receive some more precommits,
   even though we already have +2/3)
+
+## P2P settings 
+
+This section will cover settings within the p2p section of the `config.toml`. 
+
+- `external_address` = is the address that will be advertised for other nodes to use. We recommend setting this field with your public IP and p2p port. 
+- `seeds` = is a list of comma separated seed nodes. A seed node is a node that does not participate in consensus but only helps propagate peers to nodes in the networks
```

Comment:
```suggestion
- `seeds` = is a list of comma separated seed nodes that you will connect upon a start and ask for peers. A seed node is a node that does not participate in consensus but only helps propagate peers to nodes in the networks
```

Reply:
Is it really only on start? I thought it may check in with the seed here and there? (this was my assumption, so can easily be wrong)

Reply:
not only on start.. I just wanted to emphasize that you will connect to it in the absence of peers and request more of them.